README.md

```diff
@@ -10,7 +10,8 @@ Frontend for API documentation - part of the Developer Hub
 
 2. Install chrome driver if you have not done already .
 
-   2a. If using MAC OS this can be done using command ```brew cask install chromedriver```
+   2a. If using MAC OS this can be done using command ```brew install chromedriver```
```

Comment:
how come we're not using brew cask?

Reply:
brew cast does not work anymore:
```
% brew cask install chromedriver
Error: `brew cask` is no longer a `brew` command. Use `brew <command> --cask` instead.
```
but a simple brew install worked fine for me. :-)